agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4434,7 +4434,9 @@ bool slave_thread::handle_channel_preference_query(Socket *sd, ieee1905_1::CmduM
 
 /**
  * @brief Get the channel preference 
- * 
+ * Note that the channel operating class and the preference operating class has to match,
```

Comment:
(nitpick) leave an empty line before, otherwise it will be part of the 'brief'.

Maybe even better, mark is as a precondition:
```
@pre The channel operating class and the preference operating class have to match.
```